scripts/results/arrange-data.py

```diff
@@ -24,16 +24,46 @@
 

```

Comment:
General note:
Document the source code. The code, as of now, is not easy to understand.
You need to document the **important variables** and every function. You need to practice this, so take your time.
Pay attention to grammar and typos (there are a lot of grammar issues and errors in your code, like the name of the variables, comments, input and output messages, etc.)